components/NavBar.tsx

```diff
@@ -82,6 +84,25 @@ const NavBar = (): ReactElement => {
     }
   `;
 
+  const newBadgeStyle = `
+    background: ${theme.colors.warning};
+    color: ${theme.colors.primary_dark};
+    font-size: 10px;
+    line-height: 1;
+    text-transform: uppercase;
+    border-radius: 4px;
+    padding: 2px 4px;
+    font-weight: normal;
+    margin-top: -12px;
+    margin-left: 4px;
+  `;
+
+  const [showCovizu, setShowCovizu] = useState(false);
+  useEffect(() => {
+    const nextShowCovizu = localStorage.getItem('SHOW_COVIZU') === 'true';
```

Comment:
Is this a temporary test flag? will this eventually be implemented as an env-defined feature flag?

Reply:
temporary so we can turn it on in dev for testing